tck/src/main/java/org/eclipse/microprofile/openapi/tck/AirlinesAppTest.java

```diff
@@ -650,6 +650,7 @@ public void testSchema(String type) {
         vr.body("paths.'/user'.post.requestBody.content.'application/json'.schema.maxProperties", equalTo(1024));
         vr.body("paths.'/user'.post.requestBody.content.'application/json'.schema.minProperties", equalTo(1));
         vr.body("components.schemas.User.required", hasItems("id", "username", "password")); // requiredProperties
+        vr.body("components.schemas.Sex.enum", hasItems("Male", "Female", "Other"));
```

Comment:
Does this still work after the Sex->Gender rename?  Otherwise looks great to me. :)

Reply:
Done